ui/store/actions.js

```diff
@@ -1391,6 +1391,16 @@ export function removeCollectible(address, tokenID, dontShowLoadingIndicator) {
   };
 }
 
+export function checkAndUpdateCollectiblesOwnershipStatus() {
+  return async () => {
+    try {
+      await promisifiedBackground.checkAndUpdateCollectiblesOwnershipStatus();
+    } catch (error) {
+      log.error(error);
+    }
+  };
+}
```

Comment:
Since no action is dispatched here, this doesn't need to be treated as an action. It can be a normal function, like many of the other functions in this file that just call out to the background.

```suggestion
export async function checkAndUpdateCollectiblesOwnershipStatus() {
  await promisifiedBackground.checkAndUpdateCollectiblesOwnershipStatus();
}
```

^ That suggestion also removes the `try/catch` block, which ensures any errors get reported to Sentry.

This suggestion is not enough on its own though - it changes how this has to be called. It'll need to be invoked directly rather than passed to `dispatch` now.

Reply:
updated here: https://github.com/MetaMask/metamask-extension/pull/13110/commits/cf5fa0ce01ba5ef3be07cc4fcd8617f27874e141